src/stravalib/protocol.py

```diff
@@ -15,6 +15,7 @@
 import requests
 
 from stravalib import exc
+from stravalib.model import SummaryAthlete
```

Comment:
Letting the client method convert the raw response into a `SummaryAthlete` instead of here in the `protocol` module would be more consistent with how we handle other responses.